stream.go

```diff
@@ -121,9 +123,23 @@ func (s *Stream) WriteShort(status int64, data1 int64, data2 int64) error {
 	return s.Write([]Event{evt})
 }
 
-// Writes a system exclusive MIDI message to the output stream.
+// WriteSysExBytes writes a system exclusive MIDI message given as []byte to the output stream.
+// Example usage: WriteSysExBytes(portmidi.Time(), []byte{0xF0, 0x0A, 0x2F, 0x00, 0xF7}
```

Comment:
Don't provide an example in the method doc. Write an example if you'd like. See https://blog.golang.org/examples